src/redux/categories/categories.js

```diff
@@ -0,0 +1,10 @@
+const CHECKING = 'CHECK';
```

Comment:
- [ ] Good job so far From but from [ducks pattern](https://github.com/erikras/ducks-modular-redux)...` const CREATE = 'my-app/widgets/CREATE';` so you should have something like this `const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS'; `. Please fix it :+1: 

#### Find an example below :arrow_down: 

```

// Actions
const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';

// Reducer
export default function categoriesReducer(state = [], action) {
  switch (action.type) {
    case CHECK_STATUS:
      return 'Under Construction';
    default:
      return state;
  }
}

// Action creators
export function checkStatus() {
  return {
    type: CHECK_STATUS,
  };
}

```

Reply:
ok i change the way you told me that i need to put it